src/md-pages/conduct.md

```diff
@@ -14,9 +14,12 @@ This is not a legal document, members should follow the spirit of these guidelin
 1. **Please be kind and courteous**, friendly and professional. There’s no need to be mean or rude.
 1. **Do not direct message someone unless they’ve granted permission in the server** _(or you need to discuss a moderation issue with staff)_. Reactiflux is a public community. Communicate in the open.
 1. **Please use gender neutral language _(Discord display names also work great!)_.** If somebody requests you use a pronoun, please respect their wishes.
-   - We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, neurodivergence, personal appearance, body, race, ethnicity, age, religion, nationality, or other similar characteristic.
+   - We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, neurodivergence, personal appearance, body, race, ethnicity, age, religion, nationality, location, or other similar characteristic. 
+   
+   - We can all appreciate a joke, but not at the expense of others. Our staff will act againsts any member that discriminates against others in their communication.
 1. **Respect that people have differences of opinion.** Every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer.
 1. **Keep unstructured critique to a minimum.** When providing help, answer their question before making stylistic or tool suggestions.
+1. **Avoid recurring disruptive participation** If the majority of members find your interactions contributing to a negative or hostile social environment and report it as such, and if upon moderator investigation there is evidence that you have been, it will be considered a disruption an infractions will be applied.
```

Comment:
Phrasing here is slightly confusing to me, and I think I'm not happy with the "majority of members" phrasing — doesn't seem like the kind of clear-cut guidance on participation that I think is best to have in a code of conduct. Majority as defined how? Strictly speaking, this could be reasonably interpreted as "majority of the >220k accounts in the server"

Reply:
Gotcha - someone from the mod team suggested this snippet from the CoC of one of the Discord's they're in.

Would something like the following make more sense:
```
If staff finds that your interactions consistently contribute to a negative or hostile social environment, it will be considered a disruption an infractions might be applied.
```